docs/plugin-transform-runtime.md

```diff
@@ -88,17 +88,22 @@ require("@babel/core").transform("code", {
 
 ### `corejs`
 
-`boolean` or `number` , defaults to `false`.
+`false`, `2`, `3` or `{ version: 2 | 3, proposals: boolean }`, defaults to `false`.
```

Comment:
At the top of this page a note that `runtime` does not support instance methods, now it's not correct.

Too many links to deprecated `@babel/polyfill` as a global alternative.

This page does not contain a link to `core-js` documentation. However, users should know what they are using. I think that we need to add it at least on the first `core-js` entry.

Reply:
As part of this doc part I would also add a bit more clarity around how `corejs` works in conjunction with the `corejs` option from `@babel/preset-env`.

If both options are set to something other than false then the corejs imports will be duplicated. One fore preset-env and one for transform-runtime.

A bit more clarity that if you use preset-env with corejs option you should not use the same for transform runtime will help some developer understand what is going on.

This relates to some confusion from this issue [#9728](https://github.com/babel/babel/issues/9728).

The same info could be added to the preset-env doc also for reference.